src/hotspot/share/gc/shenandoah/shenandoahHeapRegion.hpp

```diff
@@ -397,7 +397,7 @@ class ShenandoahHeapRegion {
   bool oop_fill_and_coalesce();
 
   // Like oop_fill_and_coalesce(), but without honoring cancellation requests.
-  bool oop_fill_and_coalesce_wo_cancel();
+  bool oop_fill_and_coalesce_without_cancel();
```

Comment:
Along similar lines, not sure if you also got `register_object_wo_lock` in this round of cleanups.

Reply:
Yup, renamed those as well.